code/obj/submachine/slots.dm

```diff
@@ -133,229 +142,51 @@
 	else
 		. = ..()
 
-/obj/submachine/slot_machine/proc/money_roll()
-	var/roll = rand(1,1400)
+/obj/submachine/slot_machine/proc/money_roll(wager)
+	var/roll = rand(1, max_roll)
 	var/exclamation = ""
 	var/win_sound = "sound/machines/ping.ogg"
-	var/amount_text = "error credits"
 	var/amount = 0
 
-	if (roll == 1)
-		command_alert("Congratulations to [src.scan.registered] on winning the Jackpot of ONE MILLION CREDITS!", "Jackpot Winner")
+	if(wager <= 250)
+		roll = max(5, roll)
+	if(src.emagged)
+		roll = max(roll, 200)
+
+	if (roll == 1) //1
 		win_sound = "sound/misc/airraid_loop_short.ogg"
 		exclamation = "JACKPOT! "
-		amount = 1000000
-		amount_text = "MILLION CREDITS"
-	else if (roll > 1 && roll <= 5)
-		command_alert("Congratulations to [src.scan.registered] on winning a hundred thousand credits!", "Big Winner")
-		win_sound =  "sound/misc/klaxon.ogg"
-		exclamation = "Big Winner! "
-		amount = 100000
-		amount_text = "a hundred thousand credits"
-	else if (roll > 5 && roll <= 25)
+		amount = 300 * wager
+		command_alert("Congratulations to [src.scan.registered] on winning a Jackpot of [amount] credits!", "Jackpot Winner")
+	else if (roll <= 5) //4
+		command_alert("Congratulations to [src.scan.registered] on winning [amount] credits!", "Big Winner")
 		win_sound =  "sound/misc/klaxon.ogg"
 		exclamation = "Big Winner! "
-		amount = 10000
-		amount_text = "ten thousand credits"
-	else if (roll > 25 && roll <= 50)
+		amount = 100 * wager
+	else if (roll <= 15) //10 or 15
 		win_sound =  "sound/musical_instruments/Bell_Huge_1.ogg"
+		exclamation = "Big Winner! "
+		amount = 50 * wager
+	else if (roll <= 65) //50 - 500
 		exclamation = "Winner! "
-		amount = 1000
-		amount_text = "a thousand credits"
-	else if (roll > 50 && roll <= 100)
-		win_sound =  "sound/musical_instruments/Bell_Huge_1.ogg"
+		amount = 10 * wager
+	else if (roll <= 165) //100 - 500
 		exclamation = "Winner! "
-		amount = 100
-		amount_text = "a hundred credits"
-	else if (roll > 100 && roll <= 200)
+		amount = 5 * wager
+	else if (roll <= 265) //100 - 300
 		exclamation = "Winner! "
-		amount = 50
-		amount_text = "fifty credits"
-	else if (roll > 200 && roll <= 500)
-		amount = 10
-		amount_text = "ten credits"
+		amount = 3 * wager
+	else if (roll <= 715 && wager <= 250) //450
+		exclamation = "Small Winner! "
+		amount = 1 * wager
 	else
 		src.visible_message("<span class='subtle'><b>[src]</b> says, 'No luck!'</span>")
 
 	if (amount > 0)
-		src.visible_message("<span class='subtle'><b>[src]</b> says, '[exclamation][src.scan.registered] has won [amount_text]!'</span>")
+		src.visible_message("<span class='subtle'><b>[src]</b> says, '[exclamation][src.scan.registered] has won [amount] credits!'</span>")
 		playsound(src, "[win_sound]", 55, 1)
 		src.available_funds += amount
 
-/obj/submachine/slot_machine_manta
```

Comment:
Sanity check that this isn't used anywhere (e.g.: Manta)?

Reply:
no hits from searchin